stanford-earth/SE3_BLT
# EARTH-409 Adding simplesamlphp configuration files and Drupal modules.

# NOT READY
- Adds simplesamlphp files to project; configuration and testing will be done on server

# Summary
- Add acquia simplesamlphp configuration files to system and related Drupal modules to docroot
- Does not configure SimpleSAMLphp nor enables Drupal modules.

# Needed By next push to production

# Steps to Test

1. git fetch
2. git checkout EARTH-409
3. project directory should now include simplesamlphp directory
4. composer update
5. docroot/modules/contrib should now include externalauth and simplesamlphp_auth modules
